### PR TITLE
Excerise Entity and Relations

### PR DESCRIPTION
**Pull request by Throupy**
Created Exercise Entity, Configured relationships and created and registered WorkoutService.

### CODE REVIEW

File: PTWiseApp/Services/WorkoutService.cs
Comment (kacpermarciniak979) [inline PTWiseApp/Services/WorkoutService.cs:1]: Formatting on this file is scewed

File: PTWiseApp.API/Entities/Exercise.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    public class Exercise
+    {
+        [Key]
+        public int Id { get; set; }
+        public string? Name { get; set; }
+        public ExerciseType ExerciseType { get; set; } 
+        public int Sets { get; set; }
+        // Reps should be string - in case of "to failure", or similar.
+        public string Reps { get; set; }
+        public int RestPeriod { get; set; }
```
Comment (kacpermarciniak979) [inline PTWiseApp.API/Entities/Exercise.cs:15]: Specify RestPeriod variable name with units of what RestPeriod is measured in. Alternatively use TimeSpan data type

File: PTWiseApp/Services/WorkoutService.cs
Comment (Throupy) [reply PTWiseApp/Services/WorkoutService.cs:1]: You'll have to sort this - look Ok to me.

File: PTWiseApp.API/Controllers/WorkoutController.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using PTWiseApp.API.Entities;
+using PTWiseApp.API.Services;
+
+namespace PTWiseApp.API.Controllers
+{
+    [Route("api/[controller]/[action]")]
+    [ApiController]
+    public class WorkoutController : Controller
```
[the remark this answers is not in the document]
Comment (Throupy) [reply PTWiseApp.API/Controllers/WorkoutController.cs:9]: Controller inherits from ControllerBase, but will add for sake of consistency

**Pull request merged**